src/pg/counts/counts-pg-store.ts

```diff
@@ -213,6 +233,16 @@ export class CountsPgStore extends BasePgStoreModule {
     return result[0].count;
   }
 
+  private async getRecursiveCount(recursive?: boolean): Promise<number> {
+    const rec = recursive !== undefined ? [recursive] : [true, false];
+    const result = await this.sql<{ count: number }[]>`
+      SELECT COALESCE(SUM(count), 0) AS count
+      FROM counts_by_recursive
+      WHERE recursive IN ${this.sql(rec)}
+    `;
+    return result[0].count;
```

Comment:
Interesting combo of `IN []` and `[0].count`. I guess, it really doesn't matter performance-wise, since it's max 2 rows 😅